projects/plugins/jetpack/extensions/blocks/writing-prompts/writing-prompts.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Writing prompts.
+ *
+ * @since $$next-version$$
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\WritingPrompts;
+
+use Automattic\Jetpack\Blocks;
+
+const FEATURE_NAME = 'writing-prompts';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the writing prompt integration for the block editor.
+ */
+function register_extension() {
+	Blocks::jetpack_register_block( BLOCK_NAME );
+}
+
+/**
+ * Retrieve a daily writing prompt from the wpcom API and cache it.
+ *
+ * @return string JSON response from the API, as a string.
+ */
+function get_daily_writing_prompt() {
+	$today         = date_i18n( 'Y-m-d', true );
+	$transient_key = 'jetpack_writing_prompt_' . $today;
+	$prompt_today  = get_transient( $transient_key );
+
+	// Return the cached prompt, if we have it. Otherwise fetch it from the API.
+	if ( false !== $prompt_today ) {
+		return $prompt_today;
+	}
+
+	$blog_id  = \Jetpack_Options::get_option( 'id' );
+	$response = \Automattic\Jetpack\Connection\Client::wpcom_json_api_request_as_user(
+		'/sites/' . $blog_id . '/blogging-prompts?from=' . $today . '&number=1',
+		'v2',
+		array(),
+		null,
+		'wpcom'
```

Comment:
it looks like `null`, and `'wpcom'` are the default arguments anyway.

Reply:
Fair point--for me being explicit here makes the code more readable, as there's no question what API namespace is being used.